### PR TITLE
fix duplicated manifestwork apply.

### PR DESCRIPTION
**Pull request by morvencao**
Signed-off-by: morvencao <lcao@redhat.com>

### CODE REVIEW

File: pkg/cluster/manifestwork.go [hunk 1]
```diff
@@ -482,6 +482,8 @@ func ApplySubManifestWorks(ctx context.Context, kubeClient *kubernetes.Clientset
 		if err != nil {
 			return nil, err
 		}
+
+		return desiredSubscription, nil
```
Comment (clyang82) [inline pkg/cluster/manifestwork.go:486]: Do you mean that we do not do upgrade automatically?

Comment (morvencao) [reply pkg/cluster/manifestwork.go:486]: The logic here is to avoid to go to update logic when we just created the object.

**Comment by clyang82** [comment]
Good catch.
/lgtm

**Pull request merged**